index.js

```diff
@@ -32,7 +32,7 @@ class Request {
   }
 
   getHeader (name) {
-    this.getAllHeaders().get(name)
+    return this._headers && this._headers.get(name) || this.getAllHeaders().get(name)
```

Comment:
could you make this ternary instead? `this._header ? this._header.get(name) : this.getAllHeaders().get(name)`

Reply:
we can shorthand by `return this.getAllHeaders().get(name)`, in function you missing `return`